src/engraving/libmscore/property.cpp

```diff
@@ -142,7 +142,7 @@ static constexpr PropertyMetaData propertyList[] = {
     { Pid::SCALE,                   false, "scale",                 P_TYPE::SCALE,          DUMMY_QT_TR_NOOP("propertyName", "scale") },
     { Pid::LOCK_ASPECT_RATIO,       false, "lockAspectRatio",       P_TYPE::BOOL,           DUMMY_QT_TR_NOOP("propertyName", "aspect ratio locked") },
     { Pid::SIZE_IS_SPATIUM,         false, "sizeIsSpatium",         P_TYPE::BOOL,           DUMMY_QT_TR_NOOP("propertyName", "size is spatium") },
-    { Pid::TEXT,                    true,  "text",                  P_TYPE::STRING,         DUMMY_QT_TR_NOOP("propertyName", "text") },
+    { Pid::TEXT,                    true,  "text",                  P_TYPE::STRING,         DUMMY_QT_TR_NOOP("propertyName", "text") }, 
```

Comment:
did uncrustify pick this up?

Reply:
no

Reply:
it probably should have though

Reply:
@sidharth-anand please remove that stzray space at the end of the line